docs/Installing.asciidoc

```diff
@@ -356,3 +356,35 @@ KVM doesn't work
 * make sure the 'geekotest' user can access +/dev/kvm+
 * make sure you are not already running other hypervisors such as VirtualBox
 * when running inside a vm make sure nested virtualization is enabled (pass nested=1 to your kvm module)
+
+Tests try to access openqa via 'localhost' inside qemu which fails
+==================================================================
+
+You can use the environement variable +OPENQA_HOST+ from your perl test
+scripts, so they can communicate with the real host. This variable will
```

Comment:
"*the* real host" is kind of outdated with remote workers. And the information looks wrong too - as the only test using OPENQA_HOSTNAME is yast2_lan passing it to getent ahosts - which doesn't take http urls I assume